app/representers/cart_representer.rb

```diff
@@ -5,14 +5,13 @@ module CartRepresenter
   property :id, writeable: false
   property :user_id
 
-  collection :cart_items, class: CartItem do
-    property :type, getter: ->(a) {'cart_item'}, writeable: false
-    property :pricing_id
-    property :quantity
-    property :product_name, writeable: false
-    property :unit_price, writeable: false
-    property :total_price, writeable: false
-  end
+  collection :cart_items, extend: CartItemRepresenter, class: CartItem,
+    parse_strategy: ->(fragment, i, options) {
```

Comment:
Unused block argument - `i`. If it's necessary, use `_` or `_i` as an argument name to indicate that it won't be used.<br>Unused block argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.<br>Align the elements of a hash literal if they span more than one line.<br>Avoid using {...} for multi-line blocks.